### ISSUE TITLE
RegExp sur prénom et nom du formulaire user

### ISSUE THREAD

**Issue by GwilhermPoullennec** [inventaire-des-orgues/portail#353]
<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by GwilhermPoullennec** [comment]
Comme le user est une classe toute faite, j'imagine qu'il faut la surcharger ?

**Comment by LucasBerbesson** [comment]
ça dépend de ce que tu cherches à faire

**Comment by GwilhermPoullennec** [comment]
Petit détail :
A ce que les Nom et Prénoms soient dans cette casse-là, et pas NOM et PRENOM (car on perd les accents...)
Donc qu'ils commencent par une majuscule + une minuscule

**Comment by LucasBerbesson** [comment]
Dans les colonnes de l'admin ?

**Comment by GwilhermPoullennec** [comment]
Oui, pour voir correctement dans les colonnes de l’admin, et éviter des tricheries comme juste les initiales du prénom et du nom... (déjà vu !)

**Comment by LucasBerbesson** [comment]
Le mieux est d'ajouter des méthodes `clean_first_name` et  `clean_last_name` au formulaire `accounts.forms.InscriptionForm`. [cf doc](https://docs.djangoproject.com/fr/3.1/ref/forms/validation/#cleaning-a-specific-field-attribute). Cela évite de trop surcharger le modèle User qui est assez sensible. 

On pourrait aussi ajouter un petit code javascript pour forcer la casse dans `inscription.html` du genre : 
```javascript
<script>
    String.prototype.capitalize = function () {
      return this.charAt(0).toUpperCase() + this.slice(1).toLowerCase();
    }

    $('#id_first_name,#id_last_name').keyup(function () {
      var val = $(this).val().capitalize()
      $(this).val(val)
    });
  </script>
```

**Comment by LucasBerbesson** [comment]
Message d'erreur si nom en majuscules
